### ISSUE TITLE
Dropdown: once focused to the field Enter key is not displaying list

### ISSUE THREAD

**Issue by aslam-rahman** [primefaces/primereact#1929]
[x] feature request

**Current behaviour**
On Enter, simply it stays there. (User experience is not good here because the user is not able to see the complete list)

**Expected behaviour**
on Enter, it should open up the list and mouse up and down events should be supported within the list

* **PrimeReact version:**
6.3.2
